### ISSUE TITLE
Inline suppressions do not work

### ISSUE THREAD

**Issue by schijndelg** [siemens/CtrlppCheck#131]
**Describe the bug**
In our pipeline ctrlppcheck runs with the '--inline-suppr' and '--suppressions-list=<suppressions.txt>', but the inline suppressions seem only to result in extra warnings 'Unmatched suppression: badFunc_performance_delay'.

The suppression list is used to suppress general warnings as they are not relevant for the WinCC OA control language like for example uninitialized variables.
The inline suppressions are used to entice developers to explain with a comment why that code is needed.

**To Reproduce**
1. Run CtrlPPcheck with '--inline-suppr' on the following code:
```
// ctrlppcheck-suppress-begin
delay(0);
// ctrlppcheck-suppress-end

// ctrlppcheck-suppress
delay(0);

delay(0); // ctrlppcheck-suppress
```

**Expected behavior**
The above code does not give this error:
error id="badFunc_performance_delay" severity="performance" msg="It is really neccessary to use the function delay()" verbose="It is really neccessary to use the function delay()"
Nor this message:
error id="unmatchedSuppression" severity="information" msg="Unmatched suppression: badFunc_performance_delay" verbose="Unmatched suppression: badFunc_performance_delay"

**Software version and environment (please complete the following information):**
 - OS: Debian
 - WinCC OA Version: 3.19
 - Version: 'Cppcheck 2.0'
